package.json

```diff
@@ -20,5 +20,9 @@
     "redux-devtools-extension": "^2.13.2",
     "tape": "^4.8.0",
     "typings": "^2.1.1"
+  },
+  "dependencies": {
```

Comment:
These should be dev dependencies for now. also, do you still rely on apollo-fetch ? keep it : remove it.

Reply:
Removed apollo-fetch and moved the @types/react into a dev dependency
:+1: